### PR TITLE
Implement default rich menu

### PR DESCRIPTION
**Pull request by morugu**
add default rich menu feature.
For more information, see [Set default rich menu](https://developers.line.me/en/reference/messaging-api/#set-default-rich-menu)

### CODE REVIEW

**Comment by utatti** [comment]
Don't worry, I'll look into test.

File: lib/client.ts [hunk 1]
```diff
@@ -165,6 +165,20 @@ export default class Client {
       .then(res => res.richmenus);
   }
 
+  public setDefaultRichMenu(richMenuId: string): Promise<any> {
```
Comment (utatti) [inline lib/client.ts]: Return type may be `Promise<{}>` or `Promise<void>`. Please choose what you prefer.

File: lib/client.ts [hunk 2]
```diff
@@ -165,6 +165,20 @@ export default class Client {
...
+    return this.post(URL.defaultRichMenu(richMenuId));
+  }
+
+  public getDefaultRichMenuId(): Promise<string> {
+    return this.get(URL.defaultRichMenu())
+      .then(checkJSON)
+      .then(res => res.richMenuId);
+  }
+
+  public deleteDefaultRichMenu(): Promise<any> {
```
Comment (utatti) [inline lib/client.ts]: Return type may be `Promise<{}>` or `Promise<void>`. Please choose what you prefer.

File: lib/client.ts [hunk 3]
```diff
@@ -165,6 +165,20 @@ export default class Client {
...
+  public setDefaultRichMenu(richMenuId: string): Promise<any> {
+    return this.post(URL.defaultRichMenu(richMenuId));
+  }
+
+  public getDefaultRichMenuId(): Promise<string> {
+    return this.get(URL.defaultRichMenu())
+      .then(checkJSON)
+      .then(res => res.richMenuId);
+  }
+
```
Comment (utatti) [inline lib/client.ts]: Maybe it's also good to have:

```typescript
public getDefaultRichMenu(): Promise<Types.RichMenuResponse>
```

File: lib/client.ts [hunk 4]
```diff
@@ -165,17 +165,17 @@ export default class Client {
       .then(res => res.richmenus);
   }
 
-  public setDefaultRichMenu(richMenuId: string): Promise<any> {
+  public setDefaultRichMenu(richMenuId: string): Promise<{}> {
     return this.post(URL.defaultRichMenu(richMenuId));
   }
 
-  public getDefaultRichMenuId(): Promise<string> {
+  public getDefaultRichMenuId(): Promise<Types.RichMenuResponse> {
```
Comment (utatti) [inline lib/client.ts]: Oh, this change is wrong. I meant it's good to have `getDefaultRichMenu` other than `getDefaultRichMenuId`. The return type `string` was correct here.

Comment (morugu) [reply lib/client.ts]: oh, I see.

**Comment by utatti** [comment]
Never mind about the comment above. Now I think the current implement is minimal and quite enough.

**Pull request merged**